CI/travis.linux.after_success.sh

```diff
@@ -47,7 +47,7 @@ if { [ "${TRAVIS_OS_NAME}" = "linux" ] && [ "${DEPLOY}" = "deploy" ]; } ||
 
   if [ -z "${TRAVIS_TAG}" ] && [ "${public_test_build}" != "true" ]; then
     echo "== Creating a snapshot build =="
-    bash make-installer.sh "${VERSION}${MUDLET_VERSION_BUILD}"
+    ./make-installer.sh "${VERSION}${MUDLET_VERSION_BUILD}"
```

Comment:
I can't check right now, but are those files commited as executable in git?

Reply:
I can't be certain right now (running Windows 10 + MSYS2/Mingw+w64) but they *seem* to be - but that might be due to some intelligent file handling by the MSYS2 system that notes the `.sh` extension and the hash-bang marker at the start of the file. I'll take a look next time I have Linux/FreeBSD running...